pkg/tsdb/influxdb/healthcheck.go

```diff
@@ -0,0 +1,126 @@
+package influxdb
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/grafana/grafana-plugin-sdk-go/backend"
+	"github.com/grafana/grafana/pkg/tsdb/influxdb/flux"
+	"github.com/grafana/grafana/pkg/tsdb/influxdb/models"
+)
+
+const (
+	refID = "healthcheck"
+)
+
+func (s *Service) CheckHealth(ctx context.Context, req *backend.CheckHealthRequest) (*backend.CheckHealthResult,
```

Comment:
I'd like to provide unit tests but I couldn't find the best way to write them. I could use suggestions.